### DISCUSSION TITLE
How do I add a new device to the library?

### DISCUSSION THREAD

**Discussion by rickym74** [netbox-community/devicetype-library#2335] [General]
I created a new device: Citrix MPX-16000. How do I get it added to the repository?

manufacturer: Citrix
model: MPX 16000
slug: mpx-16000
default_platform: null
part_number: mpx-16000
u_height: 2.0
is_full_depth: true
airflow: front-to-rear
weight: 56.0
weight_unit: lb
console-ports:
- name: Console
  type: de-9
power-ports:
- name: PS 1
  type: iec-60320-c14
  maximum_draw: 850
- name: PS 2
  type: iec-60320-c14
  maximum_draw: 850
interfaces:
- name: Interface 0/1
  type: 1000base-t
  enabled: true
  mgmt_only: false
- name: Interface 1/1
  type: 1000base-t
  enabled: true
  mgmt_only: false
- name: Interface 25/1
  type: 25gbase-x-sfp28
  enabled: true
  mgmt_only: false
- name: Interface 25/2
  type: 25gbase-x-sfp28
  enabled: true
  mgmt_only: false
- name: Interface 25/3
  type: 25gbase-x-sfp28
  enabled: true
  mgmt_only: false
- name: Interface 25/4
  type: 25gbase-x-sfp28
  enabled: true
  mgmt_only: false
- name: Interface 25/5
  type: 25gbase-x-sfp28
  enabled: true
  mgmt_only: false
- name: Interface 25/6
  type: 25gbase-x-sfp28
  enabled: true
  mgmt_only: false
- name: Interface 25/7
  type: 25gbase-x-sfp28
  enabled: true
  mgmt_only: false
- name: Interface 25/8
  type: 25gbase-x-sfp28
  enabled: true
  mgmt_only: false
- name: Interface 100/1
  type: 100gbase-x-qsfp28
  enabled: true
  mgmt_only: false
- name: Interface 100/2
  type: 100gbase-x-qsfp28
  enabled: true
  mgmt_only: false
- name: Interface 100/3
  type: 100gbase-x-qsfp28
  enabled: true
  mgmt_only: false
- name: Interface 100/4
  type: 100gbase-x-qsfp28
  enabled: true
  mgmt_only: false
- name: LOM
  type: 1000base-t
  enabled: true
  mgmt_only: true
---

**Comment by harryajc** [comment]
Hi @rickym74 

Please look over the [CONTRIBUTING.md](https://github.com/netbox-community/devicetype-library/blob/master/CONTRIBUTING.md) which gives a guide on how to raise a PR and then we will review it.

Harry
